src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/PlayerValidation.cs

```diff
@@ -2,64 +2,84 @@
 {
     using System.Text.RegularExpressions;
 
+    using static Constants.Player;
+
     /// <summary>
     /// Player validation class.
     /// </summary>
-    public static class PlayerValidation
+    internal static class PlayerValidation
     {
+        /// <summary>
+        /// Vallidates player id.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        internal static bool IdIsInvalid(int id) =>
+            id < MIN_ID;
+
         /// <summary>
         /// Validates player first name.
         /// </summary>
         /// <param name="firstName">Player first name for validation</param>
         /// <returns>Validity of Player first name</returns>
-        public static bool ValidateFirstName(string firstName)
+        internal static bool FirstNameIsInvalid(string firstName)
```

Comment:
Why to rename it?

Reply:
Because naming didn't follow the target of method. 'ValidateFirstName' does not tell if true returned, when first name is valid, or the opposite.
'PropertyIsInvalid' returns 'true' if parameter is invalid. It's much simplier to undertand without diving in method itself (which I did in order to understand what is returns).

Reply:
if (FirstNameIsInvalid) is more understandable, than if (ValidateFirstName).
In first case you understand, that there is a check that will be called.
In second case it looks like there is action which *void* return type. If you get, that it return *bool* , it's still uncertain what will be returned, if it's valid.